### PR TITLE
Ingest Auto Refresh

### LINKED ISSUE #616: Improve rendering of Chart/Logs during ingest
**Request** 

While ingest:
1.  if user has not typed any query, auto-update the chart and log pane when new data of ingest process arrives.
2. If user has typed a query, provide a tooltip message indicating that new data has arrived and query needs to be re-run.  
  - The tooltip message will signal the user who prefers keyboard to type return key. 
- A refresh button would simplify the user action when  mouse clicks are preferred. 

The progress bar will continue to indicate information about the processing of the file and auto-update will help the user by refreshing the chart and logs in a dynamic way till the processing of file ends. 

**Description of current  behavior and possible issues:**

When opening a pcap file, the progress bar provides a good indication of the processing of the pcap file.
However, the Chart that is plot while reading packets and the initial list of logs that we render, do not change during ingest unless there is some user intervention. 
The user does not have feedback that more data/logs are available and gets the impression that data is missing, The user would have to type return key to update the screen or select the  Whole Space repeatedly, but there is no clear indication when new data is available, other than guessing from the progress bar movement.

The following screen shows the case where a single log is shown. Since there is no change, the user gets confused.

The following screen shows a case where some logs are rendered, but the Chart shows no data.

### PR DESCRIPTION
**Pull request by jameskerr**
aDVBrfBfti

fixes #616 

If the tab history is empty, this component submits the current search. This
happens the first time this tab is rendered after a file is dragged in.

As the files are ingested the backend incrememnts a snapshot_counter to
indicate that more data is available to search.

Everytime space.ingest.snapshot is incremented, this component will effectively
hit the "Enter" key for the user if all these conditions are met:

* There is only 1 entry in the tab history
* The search bar inputs are in the same state as that first entry
* The current snapshot has not yet been acknowledged

If anyone of those is not true, then the component renders a message box to
allow the user to manually refresh if they want. This box appears when

* Any of the above conditions are not met
* The current snapshot has not yet been acknowledged

The snapshot is acknowledged when any of these events happen:
* A search is submitted
* The user presses the "x" button on the message box.

### CODE REVIEW

**Comment by jameskerr** [comment]
@mason-fish thanks for the comments. Those are insightful. When I was testing, I found that the clicking the x was a bit annoying too. And I can see that if I scroll, it is also annoying. We can check to see if the user has scrolled down the page before auto refreshing.

I do like your auto-refresh checkbox suggestion as well. That explicitly says what is happening. That might be my favorite idea. Would we turn off auto-refresh for them when you start typing or scrolling? 

I think this PR gets us at least 80% there so I'm inclined to merge it as is. But I would like to keep an eye on how people like it and remember these ideas here.

File: src/css/_info-notice.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  left: 0;
+  justify-content: center;
+  height: 100px;
+}
+
+.info-notice {
+  white-space: nowrap;
+  user-select: none;
+  background: white;
+  background: $gray-5;
```
[the remark this answers is not in the document]
Comment (jameskerr) [reply src/css/_info-notice.scss:15]: I can never sneak a double background past you...

[the remark this answers is not in the document]
Comment (mason-fish) [reply src/css/_info-notice.scss:15]: 😂

**Comment by mason-fish** [comment]
@jameskerr Yeah totally, I think this is good to merge. Just wanted to share those ideas. Awesome work!

**Pull request merged**